apps/rush-lib/src/schemas/rush-project.schema.json

```diff
@@ -15,6 +15,36 @@
       "type": "string"
     },
 
+    "cacheOptions": {
+      "type": "object",
+      "properties": {
+        "disableCache": {
+          "description": "NOT RECOMMENDED. Disable caching for this project. The project will never be restored from cache. This may be useful if this project affects state outside of its folder.",
```

Comment:
`NOT RECOMMENDED` in all caps maybe isn't very useful.  (Will something bad happen? How should I decided whether to use it?)

Compare with a description like this:

> This option is only used when the cloud build cache is enabled for the repo.  You can set
> disableBuildCache=true to disable caching for a specific project.  This is a useful workaround
> if that project's build scripts violate the assumptions of the cache, for example by writing
> files outside the project folder.  Where possible, a better solution is to improve the build scripts
> to be compatible with caching.